README.md

```diff
@@ -11,8 +11,6 @@ This includes  connection plugins, such as ``network_cli``, ``httpapi``, and ``n
 
 This collection has been tested against following Ansible versions: **>=2.9.10**.
 
-For collections that support Ansible 2.9, please ensure you update your `network_os` to use the
```

Comment:
I don't know why pre-commit keeps on removing this line. :/ I've rebased to latest.